### PR TITLE
ISSUE-439: Add storage class configuration in the helm chart

### PR DESCRIPTION
**Pull request by SuryaTejaTumati**

### CODE REVIEW

**Comment by pegaautomationuser** [comment]
Starting PR-473 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-473 validation on -> integ-eks

**Review by arvasrikanth — APPROVED** [review]

**Comment by pegaautomationuser** [comment]
Starting PR-473 validation on -> integ-eks

File: charts/pega/values.yaml [hunk 1]
```diff
@@ -20,6 +20,8 @@ global:
   # as an array of comma-separated strings using the certificatesSecrets parameter.
   certificatesSecrets: []
   certificates:
+  # If a storage class to be passed to the VolumeClaimTemplates in search and stream pods, it can be specified here:
+  storageClassName: ""
```
Comment (saikarthik528) [inline charts/pega/values.yaml:24]: I think it will be better if we add this line into values.yaml of pegaSearch as well

**Comment by pegaautomationuser** [comment]
Starting PR-473 validation on -> integ-eks

File: terratest/src/test/pega/pega-tier-deployment-artifactory-certificates_test.go [hunk 2]
```diff
@@ -27,6 +27,7 @@ func TestPegaDeploymentWithArtifactoryCerts(t *testing.T) {
 					"global.provider":               vendor,
 					"global.actions.execute":        operation,
 					"installer.upgrade.upgradeType": "zero-downtime",
+					"global.storageClassName": "storage-class",
```
Comment (kishorv10) [inline terratest/src/test/pega/pega-tier-deployment-artifactory-certificates_test.go:30]: This change is not required in all the test cases. In fact, create another test case in pega-tier-deployment_test.go with storage-class and perform the required assertions. All others test need to not be touched

**Comment by pegaautomationuser** [comment]
Starting PR-473 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-473 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-473 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-473 validation on -> integ-eks

**Review by kishorv10 — APPROVED** [review]

**Comment by pegaautomationuser** [comment]
Starting PR-473 validation on -> integ-eks

**Pull request merged**